src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.h

```diff
@@ -158,6 +168,11 @@ class SmokeCoAlarmServer
         chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
         const chip::app::Clusters::SmokeCoAlarm::Commands::SelfTestRequest::DecodableType & commandData);
 
+    /**
+     * @brief Expression status record values
+     */
+    int expressedStateMask = 1;
```

Comment:
Why is this signed?  and why is it not named with "m...."?

And the description of this member does not make sense...